app/controllers/communitarian/users_controller.rb

```diff
@@ -46,6 +45,7 @@ def new
 
       user ||= User.new
       user.attributes = new_user_params
+      user.username = "user.#{SecureRandom.hex(7)}"
```

Comment:
May be we need to use the same ways with authentication via linkedin, like timestamp?

Reply:
Why? 🙂

Reply:
Well, it is temporarily username when no name provided yet.

Reply:
We can use timestamp for it, sure.

Reply:
I think it can create an error. In fact it will throttle user registrations to 1 per second. I don't think we need it.